test/unit/alignment/matrix/edit_distance_matrix/edit_distance_trace_matrix.hpp

```diff
@@ -0,0 +1,42 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <seqan3/alignment/matrix/edit_distance_trace_matrix_full.hpp>
+#include <seqan3/alignment/matrix/row_wise_matrix.hpp>
+
+#include "../../pairwise/fixture/alignment_fixture.hpp"
+
+using namespace seqan3;
```

Comment:
You are going to remove the global namespaces in the future PR.

Reply:
I'll do it in the follow up